ee-9/feature-pack/pom.xml

```diff
@@ -523,7 +523,7 @@
                 </exclusion>
                 <exclusion>
                     <groupId>org.wildfly.core</groupId>
-                    <artifactId>wildfly-elytron-integration</artifactId>
+                    <artifactId>wildfly-elytron-integration-legacy-namespace</artifactId>
                 </exclusion>
```

Comment:
This can be removed altogether. The https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/pom.xml doesn't declare this dependency.

Almost all these kinds of exclusions in this pom can be removed but that's out of scope for this!

Reply:
Removed, thanks.